### PR TITLE
docs(core): fix accessibility issues in card component

### PR DESCRIPTION
**Pull request by OleksiyDrozdov**
## Related Issue.

Closes https://github.com/SAP/fundamental-ngx/issues/6099

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [n/a] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [n/a] API boundary values
-   [n/a] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [n/a] missing API documentation or it is not understandable
-   [n/a] poor examples
-   [n/a] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [n/a] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review

-   [ ] visual misalignments/updates
-   [ ] check Light/Dark/HCB/HCW themes
-   [ ] RTL/LTR - proper rendering and labeling
-   [ ] responsiveness(resize)
-   [ ] Content Density (Cozy/Compact/(Condensed))
-   [ ] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [ ] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [ ] Mouse vs. Keyboard support
-   [ ] Text Truncation

2. API and functional correctness

-   [ ] check for console logs (warnings, errors)
-   [ ] API boundary values
-   [ ] different combinations of components - free style
-   [ ] change the API values during testing

3. Documentation and Example validations

-   [ ] missing API documentation or it is not understandable
-   [ ] poor examples
-   [ ] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts [hunk 1]
```diff
@@ -12,6 +12,6 @@ export class CardBarComponent {
     /** value in percentage, minimum 0, maximum 100 */
     @Input() value = 0;
 
-    /** bar hex color */
-    @Input() color = '#1866b4';
+    /** color name: information, warning, critical, success */
+    @Input() color: string = null;
```
Comment (platon-rov) [inline apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts:16]: Maybe enum should be used here?

File: angular.json [hunk 2]
```diff
@@ -3806,1597 +3806,1597 @@
                 }
```
Comment (platon-rov) [inline angular.json:3806]: I think this file, `ng.json` and `tsconfig.base.json` should not be changed in the scope of that pr.

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts (hunk 1 above)
Comment (platon-rov) [reply apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts:16]: Actually, these values better describe `status`, not `color`.

File: apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.ts [hunk 3]
```diff
@@ -43,13 +48,26 @@ export class CardKpiExampleComponent implements AfterViewInit {
     }
 
     drawChart(visualization: Visualization): void {
-        const data = visualization.arrayToDataTable([
+        const dataTabel = [
```
Comment (platon-rov) [inline apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.ts:51]: Typo, `dataTable`.

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <div class="bar-row">
-    <div class="bar" [style.width.%]="value" [style.backgroundColor]="color"></div>
+    <div [attr.aria-label]="status" [class]="status ? 'bar' + ' ' + status : 'bar'" [style.width.%]="value"></div>
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.html:2]:
```suggestion
    <div [attr.aria-label]="status" [class]="status ? 'bar ' + status : 'bar'" [style.width.%]="value"></div>
```

**Review by Betrozov — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
 <div class="bar-row">
-    <div class="bar" [style.width.%]="value" [style.backgroundColor]="color"></div>
+    <div [attr.aria-label]="labeli18n" [class]="status ? 'bar ' + status : 'bar'" [style.width.%]="value"></div>
```
Comment (dimamarksman) [inline apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.html:2]: `class="bar" [class.status]="!!status"`?

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.scss [hunk 6]
```diff
@@ -4,11 +4,31 @@
     align-items: center;
     width: 100%;
 
+    .test {
+        color: aqua;
+    }
```
Comment (dimamarksman) [inline apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.scss:9]: do we need `.test`?

**Review by dimamarksman — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts [hunk 7]
```diff
@@ -12,6 +14,9 @@ export class CardBarComponent {
     /** value in percentage, minimum 0, maximum 100 */
     @Input() value = 0;
 
-    /** bar hex color */
-    @Input() color = '#1866b4';
+    /** status name: information, warning, critical, success */
+    @Input() status: CardBarStatus = 'information';
+
+    /** variable for i18n */
+    @Input() labeli18n: string = this.status;
```
Comment (platon-rov) [inline apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts:21]: Let's name it `statusLabel` or so to make it more meaningful.

File: angular.json (hunk 2 above)
Comment (platon-rov) [reply angular.json:3806]: > I think this file, ng.json and tsconfig.base.json should not be changed in the scope of that pr.

Bump for this one.

**Review by platon-rov — APPROVED** [review]

**Comment by Sanjay-1312** [comment]
Issue 1: Fixed.

Issue 2: Partially fixed.
Observed Behavior: On accessing the 'Approve' and 'Reject' button, the color information such as 'Positive' and 'Negative' information is read but the Button name such as 'Approve' and 'Reject' is not read.
Expected Behavior: The name type and status of the button should be read and the information such as visible label and status of the button should be read.

For Reference.
https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.m.OverflowToolbarButton/sample/sap.m.sample.OverflowToolbarFooter

Issue 3:
Same issue persists.

Issue 4: Fixed.

Issue 5: Fixed.

Issue 6: Fixed.

issue 7:  Same issue persists.
For reference
https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.ui.integration.widgets.Card/sample/sap.ui.integration.sample.CardsLayout

Retest Details:
https://deploy-preview-6937--fundamental-ngx.netlify.app/#/core/card
JAWS 2021

**Pull request closed without merging**